components/webui/imports/ui/SearchView/SearchResults.jsx

```diff
@@ -1,9 +1,25 @@
-import React from "react";
+import React, {useCallback} from "react";
```

Comment:
```suggestion
import React from "react";
```

Reply:
Or did you want to use this for `handleLoadMoreResults`?

Reply:
Right, I forgot to remove this after modifying `handleLoadMoreResults` to call `setVisibleSearchResultsLimit` with a callback instead.